backend/tasks/runs.go

```
@@ -1,162 +1,249 @@
 
 		vcsToken, err := services.GetVCSTokenFromJob(*job, gh)
 		if err != nil {
-			log.Printf("could not get vcs token: %v", err)
+			slog.Error("could not get vcs token",
+				"error", err,
+				"jobId", job.ID,
+				runContext)
 			return
 		}
 
 		ciBackend.TriggerWorkflow(*spec, *runName, *vcsToken)
```

golangci-lint (1.64.8)

Error return value of `ciBackend.TriggerWorkflow` is not checked

(errcheck)